setup.py

```diff
@@ -99,41 +99,51 @@ def compile_native_invocation_handler(*possible_homes):
     # for android, we use SDL...
     LIBRARIES = ['sdl', 'log']
     LIBRARY_DIRS = ['libs/' + getenv('ARCH')]
+
 elif PLATFORM == 'darwin':
-    FRAMEWORK = subprocess.Popen(
```

Comment:
I think the renaming will just cause confusion. Leave it as `FRAMEWORK`, please.